patches/common/chromium/add_contentgpuclient_precreatemessageloop_callback.patch

```diff
@@ -0,0 +1,44 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <milan.burda@gmail.com>
+Date: Thu, 11 Apr 2019 14:49:20 +0200
+Subject: Add ContentGpuClient::PreCreateMessageLoop() callback
+
```

Comment:
Please add a description to the patch, preferably w/ a link to the Chromium CL once you upload it.

Reply:
Added description. I haven't tried upstreaming Chromium patches yet. I will update the patch with a CL link in case I do it in a follow up PR, is that ok?